server.R

```diff
@@ -13,70 +13,40 @@ library(shiny)
 library(reticulate)
 library(waiter)
 
+# reticulate::use_condaenv("synapse")
```

Comment:
Don't necessarily need to use `pyenv` like the instructions say, can use `conda_env` if exists.